src/WebSocketsServer.h

```diff
@@ -33,7 +33,7 @@
 
 
 
-class WebSocketsServer: private WebSockets {

```

Comment:
better use ```protected``` this way the API is hidden for the normal usage but you can still access all function when you extend the class.